### ISSUE TITLE
docker-compose fails (setuptools module missing)

### ISSUE THREAD

**Issue by pettai** [google/timesketch#1191]
Following the instructions on installing timesketch via docker-compose

`ubuntu@tfsa:~/timesketch/docker$ docker-compose up`

...ends up with this error:

```
[...]
Setting up plaso-tools (20200430-1ppa1~bionic) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Removing intermediate container dda12f0ee07c
 ---> cb515e91238d
Step 7/17 : RUN pip3 install timesketch
 ---> Running in a97cb6808c1b
Collecting timesketch
  Downloading https://files.pythonhosted.org/packages/ed/00/4fd5de4724b90719cf7777e19f5144bd6fa040333b382ced2d8ef4785d24/timesketch-20200507.tar.gz (8.9MB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ModuleNotFoundError: No module named 'setuptools'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ophpati2/timesketch/
ERROR: Service 'timesketch' failed to build: The command '/bin/sh -c pip3 install timesketch' returned a non-zero code: 1
```

The fix is trivial:
```
--- timesketch/docker/Dockerfile.orig	2020-05-13 07:38:16.231279535 +0000
+++ timesketch/docker/Dockerfile	2020-05-13 07:47:25.105866242 +0000
@@ -13,7 +13,9 @@
                        lsb-release \
                        python3-dev \
                        python3-pip \
-                       python3-psycopg2
+                       python3-psycopg2 \
+                       python3-setuptools \
+                       python3-wheel

 # Install Plaso, nodejs and yarn.
 RUN apt-get --no-install-recommends -y install software-properties-common
```

**Comment by jaegeral** [comment]
Can confirm this Bug.

**Comment by berggren** [comment]
This has been fixed, and just waiting for review/merge.

**Comment by berggren** [comment]
Hi, this has been fixed in #1207 as part of a larger refactor and fixit for docker in TS. We now build base images automatically that is used.
